lib/tasks/update_related_links_from_json.rake

```diff
@@ -0,0 +1,48 @@
+namespace :content do
```

Comment:
Might be a good idea to put all this code in a service class so it can be tested

Reply:
It's possible. Originally I wrote this script was in content-tagger and followed the existing pattern there when talking to our APIs (i.e. https://github.com/alphagov/content-tagger/blob/master/lib/tasks/load_legacy_mappings.rake).

Reply:
I agree with moving some of the logic out of the rake task - it'll make it easier down the line to turn this into an automated process.

Reply:
If it's not essential for the present, then it will be good to see if the related links test performs well. Definitely agree though.